### ISSUE TITLE
Documentation deviates from implementation

### ISSUE THREAD

**Issue by MichaelDBA** [cybertec-postgresql/pg_squeeze#7]
I just downloaded and installed 1.1 version of pg_squeeze, but when I run it based on the documentation, I get a bunch of errors for invalid column names, column types, etc.

For instance, documentation states this:
```
INSERT INTO squeeze.tables	(tabschema, tabname, first_check, task_interval, free_space_extra,
	vacuum_max_age, max_retry)
	VALUES	('public', 'bar', now(), '1 day', 30, '2 hours', 2);
```
But there is no first_check and task_interval in the squeeze.tables table, and now() is an invalid parameter type for schedule column.  Here is the ddl of the squeeze.tables that I am running against:
```
     Column      |         Type          | Collation | Nullable |                  Default
------------------+-----------------------+-----------+----------+--------------------------------------------
 id               | integer               |           | not null | nextval('squeeze.tables_id_seq'::regclass)
 tabschema        | name                  |           | not null |
 tabname          | name                  |           | not null |
 clustering_index | name                  |           |          |
 rel_tablespace   | name                  |           |          |
 ind_tablespaces  | name[]                |           |          |
 schedule         | time with time zone[] |           | not null |
 free_space_extra | integer               |           | not null | 50
 min_size         | real                  |           | not null | 8
 vacuum_max_age   | interval              |           | not null | '01:00:00'::interval
 max_retry        | integer               |           | not null | 0
 skip_analyze     | boolean               |           | not null | false

```

**Comment by ahouska** [comment]
I don't see this inconsistency with the package I got from https://github.com/cybertec-postgresql/pg_squeeze/archive/master.zip

Is anything wrong about your local git repository?

**Comment by MichaelDBA** [comment]
I got it from git repo:
```
git clone https://github.com/cybertec-postgresql/pg_squeeze.git ./pg_squeeze
cd pg_squeeze
make
make install
```

**Comment by ahouska** [comment]
I have no idea why you get the wrong README. After executing your commands, "git branch" shows that I am on the "master" branch and the INSERT statement in README looks like this:

        INSERT INTO squeeze.tables
        (tabschema, tabname, schedule, free_space_extra, vacuum_max_age,
        max_retry)
        VALUES ('public', 'bar', '{22:30, 03:00}', 30, '2 hours', 2);

Are you really looking at the README you've checked out? I wonder if you happen to read it via the github web interface and have selected the 1.0 branch.

**Comment by MichaelDBA** [comment]
I am looking at the README online for branch, REL1_0_PG_10.

Isn't that the one I should be using for PostgreSQL version 10?

**Comment by ahouska** [comment]
Your first message indicates that you try to use 1.1 (which is actually still "master", i.e. not released yet), so the corresponding README is: https://github.com/cybertec-postgresql/pg_squeeze/blob/master/README

As for the delay: I didn't consider it high priority because I had already sent you the link to get the *.zip package and you didn't complain. Unless you want to participate on development, "git clone" is the less easy way to get the code.

And as for the master inactivity: we were not sure if v1.1 should include some more features. I'm just thinking whether https://github.com/cybertec-postgresql/pg_squeeze/issues/6 should be fixed in v1.1 or v1.2. I haven't heard of any other feature request.

**Comment by MichaelDBA** [comment]
Thanks for the update.  Yes you are correct: I was not using the correct readme file for the branch I downloaded.  I did have some other problems that I will note in a new issue.
